tests/lib/gui/lvgl/boards/stm32h747i_disco_m7.overlay

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2024 STMicroelectronics
```

Comment:
Check that board names are still valid with hwmv2

Reply:
done